src/nowsecure-action.ts

```diff
@@ -9,6 +9,13 @@ import fs, { promises } from "fs";
 import { convertToSarif } from "./nowsecure-sarif";
 import { NowSecure } from "./nowsecure-client";
 import { promisify } from "util";
+import {
+  convertToSnapshot,
+  submitSnapshotData,
+  ActionContext,
+} from "./nowsecure-snapshot";
+import type { PullReportResponse } from "./types/platform";
+import * as github from "@actions/github";
```

Comment:
It appears the existing imports are alphabetically sorted by the right-hand-side, so I guess we should stick to that here.

Reply:
I'll be honest I thought that Prettier sorted import statements, but it actually doesn't. For that reason, I'll update the repo after the fact to add the plugin and apply the formatting to avoid cluttering the diff.